### PR TITLE
Daily extraction

### PR DESCRIPTION
**Pull request by megansam5**
solves issue #12 #20 

script to be run every day to move old data from RDS to S3

### CODE REVIEW

File: data-transfer-pipeline/extract.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        f'DATABASE={ENV['DB_NAME']};'
+        f'UID={ENV['DB_USER']};'
+        f'PWD={ENV['DB_PASSWORD']};'
+        'Encrypt=yes;'
+        'TrustServerCertificate=yes;'
+        # this workings in production but should be:
+        # f'TrustStore={path to pem certificate}' when deployed
+        # check how this would work in docker image
+    )
+    return conn
```
Comment (CalebMathers) [inline data-transfer-pipeline/extract.py:24]: Could just import this function from the other script as it is exactly the same

File: .gitignore
Comment (CalebMathers) [inline .gitignore:1]: This is a robust git ignore if I've ever seen one!

File: data-transfer-pipeline/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+DB_NAME=<your-database-name>
+DB_USER=<your-database-username>
+DB_PASSWORD=<your-database-password>
+AWS_ACCESS_KEY=<your-aws-access-key>
+AWS_SECRET_KEY=<your-aws-secret-key>
+BUCKET_NAME=<your-s3-bucket-name>
+SCHEMA_NAME=<your-database-schema>
+```
+
+## Running the pipeline
```
Comment (MichaelHassan) [inline data-transfer-pipeline/README.md:47]: Could also add instructions for cloud runnings , and how to dockerise the file

File: data-transfer-pipeline/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+FROM python:latest
```
Comment (CalebMathers) [inline data-transfer-pipeline/Dockerfile:1]: Will this need to be the lambda specific python to deploy to lambda?

File: data-transfer-pipeline/README.md
Comment (CalebMathers) [inline data-transfer-pipeline/README.md:1]: Great readme, definitely explains things as intended

File: data-transfer-pipeline/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# Daily Data Transfer Pipeline
+
+## Things still to do:
+- Check which columns want to be stored in s3. (at the moment only columns from recording as can link back to RDS)
```
Comment (CalebMathers) [inline data-transfer-pipeline/README.md:4]: A space here would make it uniform with the rest of the headings

**Pull request merged**